### PR TITLE
Pw/label

### PR DESCRIPTION
**Pull request by patwys**

### CODE REVIEW

File: gui/label.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import pygame
+
+
+class Label():
+    def __init__(self, x_pos, y_pos, text_input, font="comicsans", font_size=50, color ="yellow"):
```
Comment (PKurowski93) [inline gui/label.py:5]: I'm wonder shouldn't we all use hex code as a basic way to colors. it's more flexible to pick right shade of color and unification of code will looks better.

Comment (Artur-Ws) [reply gui/label.py:5]: Yes, we didn't talk about this but that will be the best solution: colors added to config.ini with clear names (like: `background_gray` or sth) and hex code as value. Please remember to pick this topic on next meeting if I forget

**Review by Artur-Ws — DISMISSED** [review]
I will approve this despite this color issue, because colors will need general refactor as they are hardcoded in some places

File: gui/label.py (hunk 1 above)
Comment (PKurowski93) [inline gui/label.py:5]: IMHO parameter `text_input` should be inside draw method. It'll be better when we dont have to import instance of player(for print HP for example) inside graphic module. When parameter `text_input` will be inside draw method instance will be imported only in one different file(Game_state).

**Review by PKurowski93 — CHANGES_REQUESTED** [review]

**Review by PKurowski93 — APPROVED** [review]

**Pull request merged**